tests/foreman/ui/test_host.py

```diff
@@ -1670,6 +1670,55 @@ def test_global_registration_with_gpg_repo_and_default_package(
     assert repo_url in result.stdout
 
 
+@pytest.mark.tier2
+def test_global_registration_upgrade_subscription_manager(
+    session, module_activation_key, module_os, module_proxy, rhel7_contenthost
+):
+    """Host registration form produces a correct registration command and
+    subscription-manager can be updated from ABI or custom repository before
```

Comment:
ABI ?

Reply:
Sorry, I meant UBI
Universal Base Images (UBI)
not
application binary interface ( ABI)
 I will just remove that.